Auth0/Telemetry.swift

```diff
@@ -45,10 +46,16 @@ public struct Telemetry {
 
     mutating func wrapped(inLibrary name: String, version: String) {
         let info = Telemetry.versionInformation()
-        let wrapped = [
+        var env = Telemetry.generateEnviroment()
+        if let libVersion = info[Telemetry.VersionKey] as? String {
+            env[Telemetry.WrappedVersion] = libVersion
+        } else {
+            env[Telemetry.WrappedVersion] =  Telemetry.NoVersion
+        }
+        let wrapped: [String: Any] = [
             Telemetry.NameKey: name,
             Telemetry.VersionKey: version,
-            Telemetry.WrappedVersion: info[Telemetry.VersionKey] ?? Telemetry.NoVersion
+            Telemetry.EnvironmentKey: env
```

Comment:
I'd be careful with a var named `env` and the string `"env"`, can often get confusing to work with both.